.rubocop_todo.yml

```diff
@@ -40,6 +40,7 @@ Metrics/AbcSize:
   Max: 20
   Exclude:
     - 'lib/vagrant-service-manager/command.rb'
+    - 'plugins/guests/redhat/cap/box_version.rb'
```

Comment:
> Condition is now required to check whether data is empty or not as from vagrant 1.9.1 this block execute twice for Windows OS only

The command is executed twice? That sounds like a serious bug. Is there something in Vagrant?